README.md

```diff
@@ -85,6 +85,24 @@ You can do that by executing `flutter pub run flutter_native_splash:create`.
 Although you can setup a bunch of features in this library, it doesn't provide a way to display animations.
 If you need a more personalized splash screen, you can edit the native code or just remove this library. 
 
+### Code generation
+
+Code generation is created using `build_runner` package.\
+To configure this package edit the `build.yaml`\
+To add new files to watch for code generation add the following lines:
+```
+targets:
+  $default:
+    builders:
+      # Previous configured builders
+      ...
+      builder_package_name:
+        generate_for:
+          # Example glob for only the Dart files under `lib/models`
+          - lib/models/*.dart
+```
+To create generated code run `clean_up.sh` under [scripts] folder or the following command: `flutter pub run build_runner build --delete-conflicting-outputs`
```

Comment:
Can you add something related the structure? O mean, for example: models are defined in.... 
Not only about the code generation, more about the project structure. 
We can do it in another pull if you prefer

Reply:
You mean different from the structure defined in the rest of the README, because there is already an explanation for model and services in the Core section.
Maybe re structuring it so that the code generation section is before the folder structure?